mlir/test/fusion/e2e/mixr-gemm-gemm/lit.local.cfg

```diff
@@ -0,0 +1,2 @@
+if not config.arch_support_mfma and not config.arch_support_wmma:
```

Comment:
Some of tests inside this folder are still applicable. just that ones that starts with `mixr-gemm-gemm-add` are not supported on Navi2x.